src/sg.c

```diff
@@ -936,6 +936,12 @@ static stlink_t* stlink_open(const int verbose) {
         return NULL;
     }
 
+#if LIBUSB_API_VERSION < 0x01000106
+    libusb_set_debug(slsg->libusb_ctx, ugly_libusb_log_level(verbose));
+#else
+    libusb_set_option(slsg->libusb_ctx, LIBUSB_OPTION_LOG_LEVEL, ugly_libusb_log_level(verbose));
+#endif
+
```

Comment:
Ah, so this is the correct condition compared to the former and no longer executed one, based on `LIBUSBX_API_VERSION`, in commit e3c76aa67315db32bd303303023d6c74c51e900b. :+1:

Reply:
correct